public/components/layout.js

```diff
@@ -135,9 +146,10 @@ class UserResource extends HTMLElement {
                     + " will note be able to gather additional information about this targeted resource."
                     + " You can supply a different URI or continue with this one.")
                 uriPreview.innerHTML = `<pre>{Not Resolvable}</pre>`
-                localStorage.setItem("userResource", JSON.stringify({"@id":target})) 
+                localStorage.setItem("userResource", JSON.stringify({"@id":target, "creator":objCreator.value}))
```

Comment:
So this is adding a new `creator` JSON object within the `userResource` JSON object, right?